NeoFS binaries should be installed in venv

Testcases require some NeoFS and Neo binaries available. It would be nice to install them inside virtualenv instead of using /usr/local/bin, as stated in README.

Maybe we could also pin a particular versions and automate the environment construction as it's done in neofs-dev-env, for example.

i'd like to push this issue

currently, test runs use both downloaded `neofs-cli` and the one from the `$PATH`. I discovered this in a very unexpected way, and it can be easily reproduced: build `neofs-cli` which insta crashes (panics) and place it into the `$PATH`. Some checks go well:
```
[INFO] Command: ./neofs-cli --config env_files/neofs-env-2024-07-03-19-23-52/ir_cli_config_lfdkwyyfru.yml control healthcheck --endpoint 'localhost:57511' --ir
```
while almost all tests fail with:
```
E           RuntimeError: Command: neofs-cli --config ll/testcases/wallet_config.yml netmap snapshot --rpc-endpoint 'localhost:43429' --wallet 'll/testcases/TemporaryDir/eb3f9859-2e3b-4976-9870-13f70f94f6b5.json'
E           Error:
E           return code: 2
E           output: panic: AAAAAAAAAAAAAAAAAAAAA
```

---

imo this is absolutely incorrect and unexpected behavior. Single instance of each app must be used. Since binaries are downloaded anyway, i expect tests to operate with them and leave `$PATH` residents alone

--- a/pytest_tests/lib/helpers/common.py
+++ b/pytest_tests/lib/helpers/common.py
@@ -39,10 +39,10 @@ WALLET_PASS = os.getenv("WALLET_PASS", "")
 
 
 # Paths to CLI executables on machine that runs tests
-NEOGO_EXECUTABLE = os.getenv("NEOGO_EXECUTABLE", "neo-go")
-NEOFS_CLI_EXEC = os.getenv("NEOFS_CLI_EXEC", "neofs-cli")
-NEOFS_AUTHMATE_EXEC = os.getenv("NEOFS_AUTHMATE_EXEC", "neofs-s3-authmate")
-NEOFS_ADM_EXEC = os.getenv("NEOFS_ADM_EXEC", "neofs-adm")
+NEOGO_EXECUTABLE = os.getenv("NEOGO_EXECUTABLE", "./neo-go")
+NEOFS_CLI_EXEC = os.getenv("NEOFS_CLI_EXEC", "./neofs-cli")
+NEOFS_AUTHMATE_EXEC = os.getenv("NEOFS_AUTHMATE_EXEC", "./neofs-s3-authmate")
+NEOFS_ADM_EXEC = os.getenv("NEOFS_ADM_EXEC", "./neofs-adm")
 
 NEOFS_ADM_CONFIG_PATH = os.getenv("NEOFS_ADM_CONFIG_PATH", os.path.join(DEVENV_PATH, "neofs-adm.yml"))
 

